Update deployment guides to use volumes (#1559)
n8n-io/n8n-docs: docs/1-0-migration-checklist.md

docker run --rm -it --user root -v ~/.n8n:/home/node/.n8n --entrypoint chown n8nio/base:16 -R node:node /home/node/.n8n
```

#### Image removal

We have removed the Debian and RHEL images. If you were using these you need to change the image you use. This shouldn't result in any errors unless you were making a custom image based on one of those images. 

#### Entrypoint change

> The entrypoint for the container has changed and you no longer need to specify the n8n command, If you were previously running `n8n worker --concurrency=5` it would now be `worker --concurrency=5`

Reviewer: ```suggestion
The entrypoint for the container has changed and you no longer need to specify the n8n command. If you were previously running `n8n worker --concurrency=5` it's now `worker --concurrency=5`
```
